packages/lib/defaultEvents.ts

```diff
@@ -1,7 +1,6 @@
 import type { Prisma, Credential } from "@prisma/client";
 
 import { DailyLocationType } from "@calcom/app-store/locations";
-import { getBookingFieldsWithSystemFields } from "@calcom/features/bookings/lib/getBookingFields";
 import slugify from "@calcom/lib/slugify";
```

Comment:
`defaultEvents` is imported client side, so avoid importing `getBookingFields` in there